test.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import json
+
+class NpEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, np.integer):
+            return int(obj)
+        elif isinstance(obj, np.floating):
+            return float(obj)
+        elif isinstance(obj, np.ndarray):
+            return obj.tolist()
+        else:
+            return super(NpEncoder, self).default(obj)
+
+# netpyne/sim/save.py#L37
+#json.dumps(data, cls=NpEncoder)
+
+a = 1
+b = 1.0 
+c = np.array(3)
```

Comment:
Shouldn't `c` be `np.zeros(1, dtype="float")` to test both np.float and np.array?

Reply:
I answer myself, maybe float is the default for numpy arrays. `a` and `b` can be deleted